qemu/tests/boot_order_check.py

```diff
@@ -71,6 +76,7 @@ def _get_pci_addr_by_devid(vm, dev_id):
             process.system("ifconfig %s down" % nic.ifname)
 
     vm.resume()
+    devices_load_timeout = params.get("devices_load_timeout", 10)
```

Comment:
should convert it to integer by int()

Reply:
It will be converted to float in utils_misc.wait_for() automatically.

Reply:
I see, I think it's more graceful when the default value is integer and the get value is string.